src/docs/platforms/javascript/angular.mdx

```diff
@@ -5,150 +5,253 @@ redirect_from:
-    if (error instanceof HttpErrorResponse) {
-      // The `error` property of http exception can be either an `Error` object, which we can use directly...
-      if (error.error instanceof Error) {
-        return error.error;
-      }
-
-      // ... or an`ErrorEvent`, which can provide us with the message but no stack...
-      if (error.error instanceof ErrorEvent) {
-        return error.error.message;
-      }
-
-      // ...or the request body itself, which we can use as a message instead.
-      if (typeof error.error === "string") {
-        return `Server returned code ${error.status} with body "${error.error}"`;
-      }
-
-      // If we don't have any detailed information, fallback to the request message itself.
-      return error.message;
-    }
-
-    // Skip if there's no error, and let user decide what to do with it.
-    return null;
-  }
+**Important:** Note your DSN. The _DSN_ (Data Source Name) tells the SDK where to send events. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in the Sentry web UI.
 
-  handleError(error) {
-    const extractedError = this.extractError(error) || "Handled unknown error";
+### Automatically Send Errors with `ErrorHandler`
 
-    // Capture handled exception and send it to Sentry.
-    const eventId = Sentry.captureException(extractedError);
+`@sentry/angular` exports a function to instantiate an `ErrorHandler` provider that will automatically send Javascript errors captured by the Angular's error handler.
 
-    // When in development mode, log the error to console for immediate feedback.
-    if (!environment.production) {
-      console.error(extractedError);
-    }
-
-    // Optionally show user dialog to provide details on what happened.
-    Sentry.showReportDialog({ eventId });
-  }
-}
+```javascript
+import { NgModule, ErrorHandler } from '@angular/core';
+import { createErrorHandler } from '@sentry/angular';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule],
-  providers: [{ provide: ErrorHandler, useClass: SentryErrorHandler }],
-  bootstrap: [AppComponent],
+  // ...
+  providers: [
+    {
+      provide: ErrorHandler,
+      useValue: createErrorHandler({
+        showDialog: true,
+      }),
+    },
+  ],
+  // ...
 })
 export class AppModule {}
 ```
 
-When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
-the interceptor doesn't modify the error captured originally.
-The same goes for writing your own API services with built-in `http` methods.
+Additionally, `createErrorHandler` accepts a set of options that allows you to configure its behavior. For more details see the `ErrorHandlerOptions` interface in [our repository](https://github.com/getsentry/sentry-javascript/blob/master/packages/angular/src/errorhandler.ts).
 
-For example, the service below makes it impossible for the SDK to extract the correct data, because the full, original error `e` is not propagated.
+## Integrating Tracing
```

Comment:
alternate title to this (which would be consistent) is "Monitor Performance", but I was uncertain if tracing is a subset of performance.

Reply:
Yes, yes it is. Updated the title then.